test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,11 +1,12 @@
 package seedu.addressbook.common;
```

Comment:
It looks like unrelated commit for another PR is getting added into this PR. Can you remove it?

Reply:
I'm not sure how to. It was not in the feature/edit branch. It's supposed to be in the main branch

Reply:
You can do so by running `git revert a77790080dd242233f6f2aa873ac7c00c981ef5a` to revert the commit.